updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,96 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-upgrade-duration"]
+= Understanding {product-title} upgrade duration
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-upgrade-duration
+
+toc::[]
+
+{product-title} upgrade duration varies based on the deployment topology. This section describes the approximate time duration of each component that follows through during the cluster update. It also explains the factors that affect the upgrade duration and the applicable {product-title} parameters.
+
+From {product-title} 4 onward, {op-system-first} is the only supported operating system (OS) for cluster upgrades. For a cluster with xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[{op-system-base} worker nodes] must be upgraded separately.
```

Comment:
I'm slightly concerned that folks will read this and think "I have non-RHCOS compute.  Does that mean my cluster updates are unsupported?".  We want to make it clear that the cluster updates are still supported, but they'll need to take extra steps to get their non-RHCOS compute updated.  I still like [my earlier][1]:

> With {product-title} upgrades, {op-system-first} lifecycle management and maintenance is part of the cluster upgrade.

For your second sentence, minor wording tweak to:

> {op-system-base} compute nodes must be xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[updated separately].

But I don't feel all that strongly about any of this, so feel free to leave it as you have it now too, or make other edits.

[1]: https://github.com/openshift/openshift-docs/pull/46241#discussion_r891636772